main/acle.md

```diff
@@ -2476,8 +2477,10 @@ compiler and it is enabled.
 
 ### Name mangling
 
-The `"default"` version is not mangled on top of the language-specific name
-mangling.
+The `"default"` version is mangled with `".default"` on top of the
+language-specific name mangling.
+All versioned functions with their mangled name are always resolvable.
```

Comment:
I suggest that you move this line to the end of the previous one, i.e. make the two form a single paragraph.